README.md

```diff
@@ -68,24 +75,24 @@ Open the zip that you downloaded from one of the links above, and drag the A32NX
 See below for the location of your Community folder:
 
 For the Microsoft Store edition AND/OR Gamepass edition:
-- Copy the "A32NX" folder into your community package folder. It is located in:
+- Copy the "A32NX" folder into your community package folder. The path is:
 `C:\Users\[YOUR USERNAME]\AppData\Local\Packages\Microsoft.FlightSimulator_<RANDOMLETTERS>\LocalCache\Packages\Community`.
 
 For the Steam edition:
-- Copy the "A32NX" folder into your community package folder. It is located in:
+- Copy the "A32NX" folder into your community package folder. The path is:
 `C:\Users\[YOUR USERNAME]\AppData\Roaming\Microsoft Flight Simulator\Packages\Community`.
 
 For the Boxed edition:
-- Copy the "A32NX" folder into your community package folder. It is located in:
+- Copy the "A32NX" folder into your community package folder. The path is:

```

Comment:
`It is located in:` sounded better imo

Reply:
Also, change the package name

Reply:
What do you mean? What is the correct package name?

Reply:
Got it, it's in the latest commit for this PR.